INFO - 2024-03-25 03:20:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-25 03:22:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-25 03:22:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-03-25 03:23:04 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=16776 in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'D:\\Project (ext...', 184)
#1 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bcpet_db', 3306, '', 0)
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `ambulat...', Array, false)
#5 D:\Project (external)\BCPetHouse\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#6 D:\Project (external)\BCPetHouse\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\Project (external)\BCPetHouse\app\Models\AmbulatoirsModel.php(48): CodeIgniter\BaseModel->findAll()
#8 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(26): App\Models\AmbulatoirsModel->getAmbulatoirAndPetList()
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->index()
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#11 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#14 {main}

Next mysqli_sql_exception: MySQL server has gone away in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bcpet_db', 3306, '', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `ambulat...', Array, false)
#4 D:\Project (external)\BCPetHouse\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 D:\Project (external)\BCPetHouse\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 D:\Project (external)\BCPetHouse\app\Models\AmbulatoirsModel.php(48): CodeIgniter\BaseModel->findAll()
#7 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(26): App\Models\AmbulatoirsModel->getAmbulatoirAndPetList()
#8 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->index()
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `ambulat...', Array, false)
#3 D:\Project (external)\BCPetHouse\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 D:\Project (external)\BCPetHouse\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 D:\Project (external)\BCPetHouse\app\Models\AmbulatoirsModel.php(48): CodeIgniter\BaseModel->findAll()
#6 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(26): App\Models\AmbulatoirsModel->getAmbulatoirAndPetList()
#7 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->index()
#8 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#12 {main}
CRITICAL - 2024-03-25 03:23:04 --> Unable to connect to the database.
Main connection [MySQLi]: MySQL server has gone away
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `ambulatoir`.*, `pp`.`name`, `pp`.`owner_name`
FROM `ambulatoir`
JOIN `pet_profile` as `pp` ON `ambulatoir`.`pet_id` = `pp`.`id`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\AmbulatoirsModel.php(48): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Ambulatoir.php(26): App\Models\AmbulatoirsModel->getAmbulatoirAndPetList()
 7 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\Ambulatoir->index()
 8 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
INFO - 2024-03-25 08:36:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-25 08:36:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-25 08:36:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-03-25 08:36:59 --> Undefined array key "date_checkup"
in APPPATH\Views\admin\rawat_inap\index.php on line 52.
 1 APPPATH\Views\admin\rawat_inap\index.php(52): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "date_checkup"', 'D:\\Project (external)\\BCPetHouse\\app\\Views\\admin\\rawat_inap\\index.php', 52)
 2 SYSTEMPATH\View\View.php(226): include('D:\\Project (external)\\BCPetHouse\\app\\Views\\admin\\rawat_inap\\index.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('admin/rawat_inap/index', [], true)
 5 APPPATH\Controllers\RawatInap.php(32): view('admin/rawat_inap/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\RawatInap->index()
 7 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RawatInap))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
INFO - 2024-03-25 08:39:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-25 08:39:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-03-25 08:39:45 --> mysqli_sql_exception: Unknown column 'rawat_inap.date_checkup' in 'field list' in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `fisiolo...', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `fisiolo...')
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `fisiolo...')
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `fisiolo...', Array, false)
#4 D:\Project (external)\BCPetHouse\app\Controllers\RawatInap.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 D:\Project (external)\BCPetHouse\app\Controllers\RawatInap.php(43): App\Controllers\RawatInap->db_fisiologis('1')
#6 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\RawatInap->detail('1')
#7 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RawatInap))
#8 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#11 {main}
CRITICAL - 2024-03-25 08:39:45 --> Unknown column 'rawat_inap.date_checkup' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `fisiologis`.*, `rawat_inap`.`date_checkup`, `rawat_inap`.`prognosa`, `rawat_inap`.`medication_plan`
FROM `fisiologis`
JOIN `rawat_inap` ON `fisiologis`.`rawat_inap_id` = `rawat_inap`.`id`
WHERE `rawat_inap`.`id` = :rawat_inap.id:', [...], false)
 2 APPPATH\Controllers\RawatInap.php(21): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\RawatInap.php(43): App\Controllers\RawatInap->db_fisiologis('1')
 4 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\RawatInap->detail('1')
 5 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RawatInap))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
